tools/wpt/browser.py

```diff
@@ -404,21 +404,23 @@ def install_webdriver(self, dest=None, channel=None, browser_binary=None):
 
         if channel == "nightly":
             path = self.install_geckodriver_nightly(dest)
-            if path is not None:
-                return path
-            else:
+            if path is None:
                 self.logger.warning("Nightly webdriver not found; falling back to release")
```

Comment:
I cannot see how we actually enter the `else` block for the release build now. Shouldn't we even better fail if the nightly build is not available?

Reply:
Oops. Fixed.

I don't think we should fail because it makes a firedrill if we ever fail to get nightly geckodriver, vs getting a few outdated results for wdspec tests.